### DISCUSSION TITLE
Is there an option to toggle AntiMicroX On and Off with a hotkey?

### DISCUSSION THREAD

**Discussion by hannibal357** [AntiMicroX/antimicrox#735] [General]
Im just wondering if i can program AntiMicroX to be enable and disabled during a gaming session...

please share if such a feature exists?

**Comment by mrbeauregarde** [comment]
there's lots of methods you could employ to do so, but not within the UI of AntiMicroX. Quit to disable, restart to enable, Taskr could do it for petesake
